src/iCalendar/DateParser.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace SRF\iCalendar;
+
+use SMWDataValueFactory as DataValueFactory;
+use SMWTimeValue as TimeValue;
+
+/**
+ * @license GNU GPL v2+
+ * @since 3.1
+ */
+class DateParser {
+
+	/**
+	 * Extract a date string formatted for iCalendar from a SMWTimeValue object.
+	 *
+	 * @since 3.1
+	 *
+	 * @param TimeValue $dataValue
+	 * @param boolean $isEnd
+	 *
+	 * @return string
+	 */
+	public function parseDate( TimeValue $dataValue, $isEnd = false ) {
```

Comment:
public function parseDate( TimeValue $dataValue, bool $isEnd = false ): string {

Reply:
Doing strict typing, I know you like it but I have not come to terms on to how to approach this without creating a piece here and there. Without a clear map of how to transform an entire "package" (it would refer here to the entire ical format and its depending classes) and not just a one class I'm a bit reluctant to added only once.